### DISCUSSION TITLE
Couldn't create 'chatwoot_dev' database. Please check your configuration.

### DISCUSSION THREAD

**Discussion by Stav-Sananes** [chatwoot/chatwoot#6270] [Q&A] [answered]
Hello guys, good evening.

When I'm trying to follow the "Project Setup" phase it requires me to run the command "bundle exec rake db:create,  but the problem I get is "could not translate host name "postgres" to address: Temporary failure in name resolution
Couldn't create 'chatwoot_dev' database. Please check your configuration.
rake aborted!".
Maybe someone knows how can I fix this problem this is where I'm stuck .

*I'm trying to run the project in localhost and not in production .*
Thanks!

**Comment by sojan-official** [comment, marked as answer]
@Stav-Sananes Could you update your .env file's `POSTGRES_HOST` value as `localhost` instead? the `postgres` address is what we use for our docker based set up.

**Reply by Stav-Sananes** [reply]
Thank you very much it was fixed

**Comment by michmosh** [comment]
updating POSTGRES_HOST=localhost did not solve the issue for me - im still getting the same error
 
**Couldn't create 'chatwoot_dev' database. Please check your configuration.**

**Reply by pranavrajs** [reply]
@michmosh, could you please verify the username and password for the database in the environment variables? The application couldn’t connect to the DB, which is likely due to the configuration provided in the environment settings.

**Comment by michmosh** [comment]
Hi,
my .env variables are as following :

POSTGRES_HOST=localhost
POSTGRES_PORT=5432
POSTGRES_DATABASE=chatwoot_dev
POSTGRES_USERNAME=postgres
POSTGRES_PASSWORD=pass
 same as in my database.yml file
